src/task_9_3.py

```diff
@@ -0,0 +1,18 @@
+"""
+Создать декоратор для функции, которая принимает список чисел.
+ Декоратор должен производить предварительную проверку данных -
+ удалять все четные элементы из списка
+"""
+def decor(funct):
+    def evenel(x):
+        l = [i for i in x if i % 2 == 0]
```

Comment:
Удалить все четные :)

```suggestion
        l = [i for i in x if i % 2 != 0]
```